### DISCUSSION TITLE
Send Scrollbar back to the top

### DISCUSSION THREAD

**Discussion by mjehrhart** [emilk/egui#1320] [Q&A]
Anyone know of a way to set the scrollbar scroll position? I have a ScrollArea which renders  about 125k rows.  The user can then click to another tab to view which may have as little as 5 rows.  Then I have to slide the scrollbar back to the top to see the rows.  Ideas???

Thanks,
Matthew

**Comment by emilk** [comment]
https://docs.rs/egui/latest/egui/struct.Ui.html#method.scroll_to_rect
